ledger-core/virtual/handlers/vcachedmemoryrequest.go

```diff
@@ -84,34 +89,40 @@ func (s *SMVCachedMemoryRequest) stepWaitResult(ctx smachine.ExecutionContext) s
 
 func (s *SMVCachedMemoryRequest) stepBuildResult(ctx smachine.ExecutionContext) smachine.StateUpdate {
 	if s.object.HeadRef().IsEmpty() {
-		s.response = &payload.VCachedMemoryResponse{
+		s.response = &rms.VCachedMemoryResponse{
+			Object:     s.Payload.Object,
+			StateID:    s.Payload.StateID,
+			CallStatus: rms.CachedMemoryStateUnknown,
+		}
+	} else {
+		s.response = &rms.VCachedMemoryResponse{
 			Object:     s.Payload.Object,
 			StateID:    s.Payload.StateID,
-			CallStatus: payload.CachedMemoryStateUnknown,
+			CallStatus: rms.CachedMemoryStateFound,
```

Comment:
Is this an intentional change of CachedMemoryStateUnknown to CachedMemoryStateFound?

Reply:
That diff may be confusing, I haven't changed it's value :)